packages/client/hmi-client/src/services/knowledge.ts

```diff
@@ -68,7 +68,6 @@ export const equationsToAMR = async (
 				return response.data.result;
 			}
 		}
-		logger.error(`Equations to AMR request failed`, { toastTitle: 'Error - Knowledge Middleware' });
```

Comment:
Those will be caught by Axios? or do we need to Throw an error?

Reply:
In this case we'd never see status != 200 (because api.ts was eating the failure case), only response == null. And in that case, `api.ts` will now log the correct error to the toaster.